templates/web/base/admin/bodies/contact-form.html

```diff
@@ -22,81 +22,74 @@ <h1>[% contact.category_display | html %]</h1>
-    <p>
-        [%
-loc("Use <strong>confirmed</strong> to indicate that this contact has been
-confirmed as correct. If you are not sure of the origin or validity of the
-contact, use <strong>unconfirmed</strong>. <strong>inactive</strong> will
-remove the category from use when reporting problems, but keep it available in
-map filters, and <strong>deleted</strong> will remove the category from there
-as well.") %]
+    <p class="form-check">
+        <input type="checkbox" name="non_public" value="1" id="non_public" [% ' checked' IF contact.non_public %]>
+        <label for="non_public">[% loc('Hide reports made in this category') %]</label>
+        <span class='form-hint'>[% loc('Use this for issues that you want to allow users to report, but for which there is no public interest in displaying the report, like requesting an extra rubbish bin at a specific address.') %]</span>
     </p>
-  </div>
-  <p>
-    <label for="state">[% loc('State') %]</label>
-    <select name="state" id="state">
-      <option value="unconfirmed"[% ' selected' IF contact.state == 'unconfirmed' %]>[% loc('Unconfirmed') %]
-      <option value="confirmed"[% ' selected' IF contact.state == 'confirmed' || contact.state == "" %]>[% loc('Confirmed') %]
-      <option value="inactive"[% ' selected' IF contact.state == 'inactive' %]>[% loc('Inactive') %]
-      <option value="deleted"[% ' selected' IF contact.state == 'deleted' %]>[% loc('Deleted') %]
-    </select>
-  </p>
 
-  <div class="admin-hint">
-    <p>
-        [% loc("Check <strong>private</strong> if reports in this category should <strong>never be displayed on the website</strong>.
-                <br>
-                Normally, categories are not private.
-                <br>
-                This is suitable for issues that you want to allow users to report to the body, but for which there is no public
-                interest in displaying the report. In the UK, we've used this for services like requesting an extra rubbish bin
-                at a specific address.") %]
+    <p class="form-check">
+        <input type="checkbox" name="disable" value="1" id="disable" data-toggle-visibility="#js-disable-form-message-box" [% ' checked' IF contact.disable_form_field %]>
+        <label for="disable">[% loc('Disable form when this category is selected') %]</label>
     </p>
-  </div>
-  <p>
-    <input type="checkbox" name="non_public" value="1" id="non_public" [% ' checked' IF contact.non_public %]>
-    <label for="non_public" class="inline">[% loc('Private') %]</label>
-  </p>
 
-  <div class="admin-hint">
-    <p>
-      [% loc("Check <strong>inspection required</strong> if reports in this category <strong>must be inspected</strong> before being sent.") %]
+    <p class="form-group form-group--indented [% 'hidden-js' IF NOT contact.disable_form_field %]" id="js-disable-form-message-box">
+        <label for="disabled-message">[% loc('Message to show when form is disabled') %]</label>
```

Comment:
These messages are likely to include links to other pages - should we tell the user they should use HTML and/or automatically upgrade plain URLs to hyperlinks somehow?